docs/src/components/landing-page/MainLayoutHeader.vue

```diff
@@ -27,13 +30,14 @@
           </q-btn>
         </div>
         <q-form v-if="isSearchFieldActive" autocapitalize="off" autocomplete="off" spellcheck="false" class="search-form position-relative">
-          <q-input ref="searchInputRef" autofocus v-model="searchTerms" dark dense square debounce="300" @keydown="onSearchKeydown" @focus="onSearchFocus" @blur="onSearchBlur" placeholder="Search Quasar v2...">
+          <q-input ref="searchInputRef" autofocus v-model="searchTerms" :dark="!dark" dense square debounce="300" @keydown="onSearchKeydown" @focus="onSearchFocus" @blur="closeSearchForm" placeholder="Search Quasar v2...">
             <template #append>
-              <q-icon name="cancel" color="lp-primary" @click.stop="toggleSearchInputField" />
+              <q-icon v-if="!searchTerms" name="search" size="sm" color="lp-primary" />
+              <q-icon v-else name="cancel" @click.stop="resetSearch" class="cursor-pointer"/>
             </template>
           </q-input>
           <div class="search-result-field">
-            <q-scroll-area dark class="bg-dark text-white rounded-borders search-result-container">
+            <q-scroll-area dark class="bg-dark text-white rounded-borders search-result-container" style="width: 300px">
```

Comment:
You already have a `search-result-container` class, why not setting `width` on that class?
Never use inline style, unless the depends from the model, and in those cases you can use v-bind in style tag anyway

Reply:
This was a test I missed to eliminate during review, the weight had already been handled.